starknet_py/net/models/typed_data.py

```diff
@@ -0,0 +1,37 @@
+"""
+TypedDict structures for TypedData
+"""
+
+from typing import Dict, List, Union
+
+from typing import TypedDict
+
+
+class Parameter(TypedDict):
+    """
+    TypedDict representing a Parameter object
+    """
+
+    name: str
+    type: str
+
+
+class StarkNetDomain(TypedDict):
+    """
+    TypedDict representing a StarkNetDomain object
+    """
+
+    name: str
+    version: str
+    chainId: Union[str, int]
+
+
+class TypedData(TypedDict):
+    """
+    TypedDict representing a TypedData object
+    """
+
+    types: Dict[str, List[Parameter]]
+    primary_type: str
+    domain: StarkNetDomain
+    message: dict
```

Comment:
I believe this would be `Dict[Str, Any]` but I'm not sure about this.

Reply:
I meant the `message` parameter 😅. Types was typed correctly with `List[Parameter]`.